syntaxes/ion.tmLanguage.json

```diff
@@ -51,7 +51,7 @@
 							"name": "punctuation.definition.string.begin.ion"
 						}
 					},
-					"end": "[\"\\n]",
+					"end": "\"",
```

Comment:
can we add tests for the cases that caused the bugs and are fixed because of this PR?

Reply:
Sure